packages/utils/src/formatDate.ts

```diff
@@ -4,26 +4,40 @@ import { nb, enUS } from "date-fns/locale"
 const DEFAULT_DAYS_RELATIVE_THRESHOLD = 3
 const DEFAULT_LOCALE = "nb-NO"
 
-const dateFNSLocaleFromIntlLocale = {
+const DateFnsLocaleMap = {
   "nb-NO": nb,
   "en-US": enUS,
 } as const
 
-type SupportedLocales = keyof typeof dateFNSLocaleFromIntlLocale
+type SupportedLocales = keyof typeof DateFnsLocaleMap
+
+const DEFAULT_INTL_DATE_FORMAT_OPTIONS = { day: "2-digit", month: "2-digit", year: "numeric" } as const
+const DEFAULT_INTL_WEEKDAY_FORMAT_OPTIONS = { weekday: "long" } as const
+const DEFAULT_INTL_TIME_FORMAT_OPTIONS = { hour: "2-digit", minute: "2-digit" } as const
+
+export const IntlFormats = {
+  Date: DEFAULT_INTL_DATE_FORMAT_OPTIONS,
+  Time: DEFAULT_INTL_TIME_FORMAT_OPTIONS,
+  Weekday: DEFAULT_INTL_WEEKDAY_FORMAT_OPTIONS,
+  DateTime: { ...DEFAULT_INTL_DATE_FORMAT_OPTIONS, ...DEFAULT_INTL_TIME_FORMAT_OPTIONS },
+  DateWeekday: { ...DEFAULT_INTL_DATE_FORMAT_OPTIONS, ...DEFAULT_INTL_WEEKDAY_FORMAT_OPTIONS },
+  DateTimeWeekday: {
+    ...DEFAULT_INTL_DATE_FORMAT_OPTIONS,
+    ...DEFAULT_INTL_TIME_FORMAT_OPTIONS,
+    ...DEFAULT_INTL_WEEKDAY_FORMAT_OPTIONS,
+  },
+} as const
```

Comment:
Enum?

Reply:
I'm used to using "as const" over enums. They're lighter and easier to work with in my experience (mostly type-wise). I could move to an enum if you prefer? I see that both are used in the repo